include/wx/generic/grid.h

```diff
@@ -88,6 +88,20 @@ enum wxGridRenderStyle
                           wxGRID_DRAW_BOX_RECT
 };
 
+// Flags to pass to wxGrid::RefreshArea()
```

Comment:
We still need to decide whether this is going to be public (and then it needs to be documented) or not (and then it should be moved to a private header, which will be simpler to do if we remove the single area overload).

Reply:
Sorry again! I think the enum doesn't need to be public and I'll move it to the private header again.